<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки фигур</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fefefe;</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <property name="spacing">
     <number>16</number>
    </property>

    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Выбор стиля фигур</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #1e3a5f; background-color: #dbeafe; padding: 10px; border-radius: 12px; border: 1px solid #93c5fd;</string>
      </property>
     </widget>
    </item>

    <!-- выбор стиля фигур -->
    <item>
     <widget class="QComboBox" name="pieceStyleComboBox">
      <property name="styleSheet">
       <string notr="true">QComboBox { padding: 6px; border-radius: 6px; border: 1px solid #d1d5db; font-size: 14px; }</string>
      </property>
      <item>
       <property name="text">
        <string>classic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>basics</string>
       </property>
      </item>
     </widget>
    </item>

    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>

    <!-- кнопки -->
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <property name="spacing">
       <number>10</number>
      </property>

      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    color: white;
    background-color: #34d399;
    border: 1px solid #10b981;
    border-radius: 10px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover { background-color: #6ee7b7; }
QPushButton:pressed { background-color: #10b981; }
         </string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="backButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    color: white;
    background-color: #60a5fa;
    border: 1px solid #3b82f6;
    border-radius: 10px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover { background-color: #93c5fd; }
QPushButton:pressed { background-color: #3b82f6; }
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
